src/lib/Exception/Invalid.php

```diff
@@ -0,0 +1,8 @@
+<?php
+namespace CMPayments\Exception;
```

Comment:
Exceptions for this package should be namespaced into "cmpayments\iban\exceptions" to avoid namespace clashes with possible other cmpayments packages